testdata/go/v3/memcached-operator/monitoring/metrics/metrics.go

```diff
@@ -0,0 +1,21 @@
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"sigs.k8s.io/controller-runtime/pkg/metrics"
+)
+
+// MemcachedNumberOfReplicasReconcileCountTotal counts Memcached replicas number reconciliations
+var (
+	MemcachedNumberOfReplicasReconcileCountTotal = prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Name: "memcached_number_of_replicas_reconcile_count_total",
+			Help: "Total count of Memcached replicas number reconciliations",
+		},
+	)
+)
+
+// Register MemcachedReplicasUpdatesCountTotal with the global prometheus registry
```

Comment:
This comment should state that the function registers all metrics, and not `MemcachedReplicasUpdatesCountTotal`

Reply:
Done